resources/newssite/news-site-css/src/css/toggle.css

```diff
@@ -27,31 +27,34 @@
     width: 42px;
     height: 16px;
     display: flex;
-    background-color: var(--color-utils-gray-lighter);
+    background-color: var(--theme-toggle-background);
     border-radius: 22px;
     cursor: pointer;
+    border: var(--theme-medium-border);
 }
 
 .toggle-container .switch {
     position: absolute;
     left: 0;
     top: 3px;
-    background-color: var(--color-utils-gray);
+    background-color: var(--theme-toggle-foreground);
     border-radius: 22px;
     transition: var(--transition-default);
     width: 45%;
     height: 16px;
     pointer-events: none;
+    border: var(--theme-medium-border);
 }
 
 .toggle-container .switch::before {
     position: absolute;
     content: "";
     height: 22px;
     width: 22px;
-    left: 0;
-    bottom: -3px;
-    background-color: var(--color-utils-red);
+    top: 50%;
+    left: calc(var(--theme-border-width-medium) * -1);
+    transform: translateY(-11px);
```

Comment:
I find this a bit hacky, is there another way to do it? Maybe using `calc` with `top`?

Reply:
will use calc!